front/lib/plans/usage/types.ts

```diff
@@ -7,7 +7,12 @@ export const REPORT_USAGE_METADATA_KEY = "REPORT_USAGE";
 
 export class InvalidReportUsageError extends Error {}
 
-export const SUPPORTED_REPORT_USAGE = ["MAU_1", "MAU_5", "MAU_10"] as const;
+export const SUPPORTED_REPORT_USAGE = [
+  "MAU_1",
+  "MAU_5",
+  "MAU_10",
+  "SEAT_MEMBER",
```

Comment:
what about naming this just "SEATS" or "PER_SEAT"? I fear the "_MEMBER" suffix adds confusion since "SEAT_MEMBER" I have trouble parsing the meaning (what does it mean? if it's just seat why name it seat? if it's not just seat do we only count members but not builders?), plus in function names we sometimes use "seatMembers" sometimes just "seats"  => would feel better if we used only "seats"